src/validation/validation.cpp

```diff
@@ -2875,9 +2875,7 @@ bool DisconnectTip(CValidationState &state, const Consensus::Params &consensusPa
             {
                 CTxInputData txd;
                 txd.tx = ptx;
-                txd.nodeId = -1;
                 txd.nodeName = "rollback";
-                txd.whitelisted = false;
```

Comment:
CTxInputData does not have an initializer, so every field must be set.  Probably these were accidentally removed during a merge since they seem unrelated to the topic of this PR.